pwalvarado/counting_vowels.rb

```diff
@@ -0,0 +1,54 @@
+require 'benchmark'
+
+class SequentialVowels
+
+  def self.run
+    sequential = new
+    time = Benchmark.measure { sequential.sequential_parse }
+    print "Sequential counting \n #{sequential.vowels} \nrun in #{time.real} secs"
+
+    threaded = new
+    time = Benchmark.measure { threaded.threading_parse }
+    "\n\nParallel counting \n #{threaded.vowels} \nrun in #{time.real} secs"
+  end
+
+  def vowels
+    @vowels ||= {"a" => 0, "e" => 0, "i" => 0, "o" => 0, "u" => 0 }
+  end
+
+  def sequential_parse
+    open_file.each_slice(4000) do | batch_lines |
+      line = batch_lines.join.downcase
+      vowels.each { |k, v| vowels[k] += line.scan(k).count }
+    end
+  end
+
+  def threading_parse
+    ary = []
+    open_file.each_slice(4000) do | batch_lines |
+      ary.push(Thread.new{
+        sum = Hash.new(0)
+        line = batch_lines.join.downcase
+        vowels.each { |k, v| sum[k] = line.scan(k).count }
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.